packages/backend/src/open_payments/auth/grant.ts

```diff
@@ -18,7 +18,8 @@ export enum AccessType {
 export enum AccessAction {
   Create = 'create',
   Read = 'read',
-  Update = 'update'
+  Update = 'update',
+  List = 'list'
```

Comment:
```suggestion
  Update = 'update'
```

Reply:
I think we could keep that since we will have the `list` access action eventually.